tox.ini

```diff
@@ -57,10 +58,12 @@ passenv = *
 commands =
   pip wheel --progress-bar off -w {homedir}/.wheelhouse -f {homedir}/.wheelhouse -e .[devel_ci]
   pip install --progress-bar off --find-links={homedir}/.wheelhouse --no-index -e .[devel_ci]
+  env_kubernetes: pip install boto3
+  env_kubernetes:  pip install moto
```

Comment:
Do we not want boto3 and moto always? Also aren't these already installed as test_requires from setup.py? Why do we need to specify them directly here? (My tox is hazy, so there may be a reason)